### PR TITLE
Use tls redirection for http.

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**

### CODE REVIEW

File: config/route.yaml [hunk 1]
```diff
@@ -9,6 +9,7 @@ metadata:
 spec:
   tls:
     termination: edge
+    insecureEdgeTermination: Redirect
```
Comment (gabemontero) [inline config/route.yaml:12]: @adambkaplan this is what we talked about in some meeting wrt comparing with what sandbox was doing, correct?

Comment (adambkaplan) [reply config/route.yaml:12]: Correct.

**Comment by gabemontero** [comment]
let's see if the build on pull request fails like it did for @enarha in his PR

**Comment by enarha** [comment]
It is the same error.

**Comment by gabemontero** [comment]
> let's see if the build on pull request fails like it did for @enarha in his PR

yep same error ... either some sort of repo configuration error that I don't understand, or the creds on the prod cluster are out of whack

by comparison I see the build-service's PRs are able to push images ... see https://console-openshift-console.apps.stone-prd-rh01.pg1f.p1.openshiftapps.com/k8s/ns/tekton-ci/tekton.dev~v1beta1~PipelineRun/build-service-pull-request-4xtr5/logs/build-container before it gets pruned @AndrienkoAleksandr @enarha @adambkaplan 

I see our PRs are running in the `pkumari-tenant` namespace and the build-service PR ran in the `tekton-ci` namespace 

I bet that has something to do with it

**Comment by enarha** [comment]
That pkumari-tenant also caught my eye. Fishy.  On a different topic, one of the RHTAP tests was skipped here (it ran on my PR) with message `User AndrienkoAleksandr is not allowed to run CI on this repo.` @AndrienkoAleksandr FYI.

**Comment by gabemontero** [comment]
I've started https://redhat-internal.slack.com/archives/C032EJ007C0/p1681133905293249 for the namespace question

**Comment by gabemontero** [comment]
ok green build-on-pull-request that is back on `tekton-ci` ns

**Review by adambkaplan — APPROVED** [review]
/lgtm

**Pull request merged**